CONTRIBUTING.md

```diff
@@ -76,6 +76,13 @@ If using `pip` and `virtualenv` (not recommended), you need to `cd` to the direc
 $ pip install -e .
 ```
 
+Finally, you need to install the core dependencies in the subrepos.
+This will install `spyder`, `spyder-kernels`, `qdarkstyle`, and `python-lsp-server` in editable mode.
```

Comment:
I think QtConsole is missing in this description. A subrepo for QtConsole was added at PR https://github.com/spyder-ide/spyder/pull/17559